lib/datacrafts_io_skeleton.rb

```diff
@@ -20,12 +20,18 @@ class Composer < ::Thor
     include OptionsExtractor
     include Helpers
 
-    desc "create APP_NAME", "Create you own best Rails/JS app"
+    desc "create APP_NAME", <<~DESC
+      Create your own best Rails/JS app.
+      If you have some issues with creation please use VERBOSE=true
+    DESC
 
     def create(app_name)
-      say "Skeleton in creating your best Rails/JS application \n"
+      say <<~DESC
+        Skeleton in creating your best Rails/JS application.
+        If you have some issues with creation please use VERBOSE=true
```

Comment:
Should we try to add checks to avoid some type of common errors?

Reply:
currently it's tested manually and I've added this comment for situations when you have issues with your tools
sometimes local ruby or node versions or some libs has conflicts with our tool so not sure that we can test it
anyway sure we'll write specs for skeleton, but I was thinking in next release